chain/cosmos/cosmos_chain.go

```diff
@@ -1183,3 +1184,15 @@ func (c *CosmosChain) VoteOnProposalAllValidators(ctx context.Context, proposalI
 	}
 	return eg.Wait()
 }
+
+// GetTimeoutHeight returns a timeout height of 1000 blocks above the current block height.
+// This function should be used when the timeout is never expected to be reached
+func (c *CosmosChain) GetTimeoutHeight(ctx context.Context) clienttypes.Height {
+	height, err := c.Height(ctx)
+	if err != nil {
+		c.log.Error("failed to get chain height", zap.Error(err))
```

Comment:
[logs should begin with a capital letter](https://github.com/strangelove-ventures/interchaintest/blob/main/docs/logging.md#usage)